src/main/java/org/opentripplanner/routing/api/request/preference/BikePreferences.java

```diff
@@ -45,6 +46,8 @@ private BikePreferences() {
     this.parkCost = 120;
     this.optimizeType = BicycleOptimizeType.SAFE;
     this.optimizeTriangle = TimeSlopeSafetyTriangle.DEFAULT;
+    // very high reluctance to carry the bike up/down a flight of stairs
+    this.stairsReluctance = 100;
```

Comment:
Would you bike for 1h40m to avoid walking the stairs for 1 minute? I think 10 probably are better. We should try to use sensible values - if not, we undermine what "reluctance" mean.

Reply:
I was thinking more like: cycling 100 meters to avoid a one meter stairs. I would definitely take the long way.

Am I thinking about it the wrong way?

Reply:
We usually use time as a unit, if not it you must adjust when the speed changes. Generaflized-cost is equivalent of spending 1 second on a transit vehicle.

Reply:
I think a reason for this is that by [default the time and cost of switching between walking and biking is 0](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/src/main/java/org/opentripplanner/routing/api/request/preference/BikePreferences.java#L41-L42), we should have some more reasonable defaults. That way the multiplier here can be more realistic

Reply:
Functionally what is the unit equivalent for this type? We want it to be a "Reluctance" - than it is easier for the user to reason about it - and easier to document. We have had examples in the past where we had to change the unit of a parameter an that is costly  - so we should make it right from the start.

Reply:
It is a cost multiplier just like the other reluctances which is proportional to the length of the edge to avoid. 

So a stairs with a single step might be used more easily and a long flight of stairs that is 15 meters long will be avoided unless there really is not other option.

Should I be using `Units.reluctance` to make it easier to read?

Reply:
Or are you talking about the bike switch cost/time?

Reply:
I think is correct, multiplying the `weigh` is a reluctance. Strictly is a scalar, but the `weight` in `transit seconds`. See [StreetEdge](#src/main/java/org/opentripplanner/street/model/edge/StreetEdge.java)

Reply:
Should we have a separate PR, where we change the default switch time & cost?

Reply:
Yes, that's a good idea.